README.md

```diff
@@ -1,57 +1,53 @@
-# How to run this boilerplate
-> A tutorial on how to create mongodb and cloudinary > accounts, install dependencies and run the project.
+# Projeto: Squad Shellyda </>
 
-## 1. Install dependencies
-Run `yarn install` or `npm install` on both client and server folders, this command will install the project's dependencies.
+## Quem é nosso cliente? 
 
-## 2. Create MongoDB cluster and get the url
-1. If you don't have one, [create a mongodb account](https://www.mongodb.com/cloud), then create the cluster for this project.
+É a iJunior, empresa júnior do departamento de Ciências da Computação da UFMG, que desenvolve dois tipos de soluções digitais: **iJunior Start** e **iJunior Loop**.
 
-2. On the cluster's screen, go to **Database Access > Database Users > Add New Database User**, then create the user (remember the username and password, we're going to need them later).
+## E o nosso projeto? 
 
-3. Now go to **Network Access > Ip Whitelist > Add IP Adress > Current Ip Adress**, to add your current ip address to the cluster's whitelist, you will need to do this with every source that will access your database.
+Nosso trabalho é desenvolver um OnePage para apresentar o produto **iJunior Loop**, que é uma solução voltada para a criação de Aplicações Web com desenvolvimento em módulos. O objetivo principal do projeto é a captação de clientes, e ele deve apresentar o projeto de forma objetiva e que traga valor ao usuário.
 
-4. The mongo url will be:
+## E quais são as tecnologias utilizadas?
 
-    mongodb://[USER]:[PASSWORD]@[SHARD_URL]/[DB_NAME]?ssl=true&replicaSet=[SHARD_NAME]&authSource=admin&retryWrites=true&w=majority
+- JavaScript
+- HTML
+- CSS
+- MongoDB
+- Keystone
+- Node Js
 
-## 3. Create cloudinary account
-1. Go to [cloudinary images website](https://cloudinary.com/) and create an account.
+## Como vamos rodar o projeto?
 
-2. Under the **Account Details** section is a url named **API environment variable** this is your cloudinary url.
+O projeto foi dividido em algumas etapas. São elas:
 
-## 4. Create a env file
-Create a file named `.env` and, inside of it, place this:
+1. Produção do *wireframe*, seguida por sua validação;
+2. Produção do *mockup de alta fidelidade*, seguida por sua validação;
+3. *Sprint 1 de desenvolvimento*, onde será feita a estruturação do back-end;
+4. *Sprint 2 de desnvolvimento*, onde será feita a componentização do front-end;
+5. *Sprint 3 de desenvolvimento*, cujos objetivos são integração e deploy;
+6. *Entrega final do projeto*;
 
-    PORT=[PORT]
-    MONGO_URI=[MONGO_URI]
-    COOKIE_SECRET=[COOKIE_SECRET]
-    CLOUDINARY_URL=[CLOUDINARY_URL]
+As atividades são divididas entre os membres, que se encontram diariamente para alinhamento no projeto. Cada issue possui prazo e é marcada por tags específicas (por ordem de prioridade por exemplo), e a organização é pelo Notion e GitHub! 
```

Comment:
Hi Carol! Congratulations on your work, you're awesome! <3 

Just some details, this Readme should be written in English, sorry if didn't make it clear, and in the part "How are we going to run the project?" referred to how to start the project technically, like Edgar's description here:

**Running in development**
To run this project in development mode, we will need to run two servers, the react one on /client and keystone on /server.

The command to run react is yarn start or npm start depending on which tool was used on installation, the react server will run on port 3000 by default.

Before running the keystone server, go to /server/updates/0.0.1-admin.js and change the admin user as you want, this user will be the first created, but you will be able to create others and delete this one later.

To run keystoneJS server, use the command node index.js, the server will run on whatever port is in the variale in the env file, you will find the admin interface in http://localhost:[PORT]/admin

**Running in production**
To run the server in production, go to /client and run the command yarn server, this command will create a react production optimized build and move it to /server.

Then go to /server and run node index.js, you will find the project on http://localhost:[PORT]

-----------------------
It's really cool that you brought the information about the sprints, I hadn't thought of that way! Congratulations!

No need to undo this part, just add the technical description as well. Thanks :)